src/include/network/services/dns.rb

```diff
@@ -430,7 +431,7 @@ def InitDhcpHostname(_key)
         fix_dhclient_warning(LanItems.invalid_dhcp_cfgs)
 
         selected = NO_CHANGE_LABEL
-        items << [Item(Id(NO_CHANGE_LABEL), _("keep current settings"), true)]
+        items << Item(Id(NO_CHANGE_LABEL), _("keep current settings"), true)
```

Comment:
Ah this was the one pointed during meeting. I didn't check that yet :-/ Good to know that it is going to be fixed too ;-)